packages/react-components/src/components/FileDownloadCards.tsx

```diff
@@ -161,12 +163,18 @@ export const _FileDownloadCards = (props: _FileDownloadCards): JSX.Element => {
   const downloadFileButtonString = useMemo(
     () => () => {
       return props.strings?.downloadFile ?? localeStrings.downloadFile;
-      // Return download button without aria label
-      return props.strings?.downloadFile ?? '';
     },
     [props.strings?.downloadFile, localeStrings.downloadFile]
   );
 
+  const fileCardGroupDescription = useMemo(
+    () => () => {
+      const fileGroupLocaleString = props.strings?.fileCardGroupMessage ?? localeStrings.fileCardGroupMessage;
+      return _formatString(fileGroupLocaleString, { fileCount: `${fileMetadata.length}` });
```

Comment:
If there are 1 attachedimage and 2 attached file it will announce there is 3 attachments.
Maybe do a filter `filemetadata.filter(file => file.attachmentType === 'fileSharing').length` to get only attachments

Reply:
Good catch!